A58-client-side-weighted-round-robin-lb-policy.md

```diff
@@ -125,20 +125,20 @@ In all languages, when a subchannel state is updated to `IDLE`, the LB policy im
 
 ### Subchannel Weights
 
-To enforce `blackout_period` and `weight_expiration_period`, the LB policy tracks two timestamps along with the weight. `last_updated` tracks the last time a report with non-zero `cpu_utilization` and `qps` was received. `non_empty_since` tracks when the first non-zero load report was received, and it is reset when a subchannel comes back to `READY` or the weight expires so that it can be updated with the next non-zero load report. The blackout and expiration are enforced when the weight value is looked up.
+To enforce `blackout_period` and `weight_expiration_period`, the LB policy tracks two timestamps along with the weight. `last_updated` tracks the last time a report with non-zero `application_utilization` and `qps` was received. `non_empty_since` tracks when the first non-zero load report was received, and it is reset when a subchannel comes back to `READY` or the weight expires so that it can be updated with the next non-zero load report. The blackout and expiration are enforced when the weight value is looked up.
```

Comment:
Even if the `application_utilization` field is zero, if the `cpu_utilization` field is non-zero, then it still updates `last_updated`.  So we need to refer to both fields here.

Reply:
I moved what you suggested below to here to explain how the utilization (without quotes) is loaded once first, then updated to use the term utilization everywhere. I think this is simpler to follow later in text and code, but let me know what you think.

Reply:
+@dfawley  as well, this is the only section we explain on load metrics and we explain how the utilization is acquired first now, PTAL.